src/main/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/courtsentencing/CourtSentencingMigrationService.kt

```diff
@@ -96,7 +96,7 @@ class CourtSentencingMigrationService(
     offenderNo: String,
     context: MigrationContext<PrisonerId>,
   ): List<CourtCaseResponse> {
-    nomisCourtCases.any { it.combinedCaseId != null }.let {
+    if (nomisCourtCases.any { it.combinedCaseId != null }) {
       val caseIds = nomisCourtCases.filter { it.combinedCaseId != null }.map { it.id }
       log.info(
```

Comment:
Is this a temp method? If not you could improve things a bit by setting caseIds first then do 'if(caseIds.isEmpty()) ... '

Reply:
yes it is temporary